tests/acceptance/features/apiSpacesDavOperation/fileVersionByFileID.feature

```diff
@@ -58,3 +58,35 @@ Feature: checking file versions using file id
     And using new DAV path
     When user "Brian" tries to get the number of versions of file "/text.txt" using file-id path "/meta/<<FILEID>>/v"
     Then the HTTP status code should be "403"
+
+
+  Scenario Outline: check the file versions of a file after moving to a shared folder inside a project space as editor/viewer
```

Comment:
```suggestion
  Scenario Outline: check the versions of a file after moving to a shared folder inside a project space as editor/viewer
```

Reply:
would be great to have consistency as early as possible. See scenario below